src/scripts/components/overlay/overlay.styl

```diff
@@ -14,11 +14,24 @@
   opacity: 0.9
 
 .overlayContent
-  padding-top: 20%
+  display: flex
+  justify-content: center
+  align-items: center
   width: 60%
-  height: 80%
+  height: 70%
 
 .closeButton
   position: absolute
   top: emCalc(24px)
   right: emCalc(32px)
+
+.logo
```

Comment:
Should the logo be always on every overlay? My first thought about this was that it should be implemented in the specific overlays content